public class ArithmeticExceptionExample {
    public static void main(String[] args) {
        try {
            int result = 10 / 0;
        } catch (ArithmeticException e) {
            System.out.println("Divided by Zero Exception: " + e);
        }

        try {
            double result = Math.log(-1);
        } catch (ArithmeticException e) {
            System.out.println("Logarithm of Negative or Zero Exception: " + e);
        }

        try {
            double result = Math.tan(Math.toRadians(90));
        } catch (ArithmeticException e) {
            System.out.println("Tan 90 Degree Exception: " + e);
        }

        try {
            double result = Math.pow(0, 0);
        } catch (ArithmeticException e) {
            System.out.println("Zero Power Zero Exception: " + e);
        }
    }
}
